homeassistant/components/switchbot_cloud/const.py

```diff
@@ -5,4 +5,9 @@
 
 DOMAIN: Final = "switchbot_cloud"
 ENTRY_TITLE = "SwitchBot Cloud"
-SCAN_INTERVAL = timedelta(seconds=600)
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=600)
+DEVICE_SCAN_INTERVAL = {"MeterPlus": timedelta(seconds=60)}
```

Comment:
There is an issue about increasing the scan interval, you can find more explanation about that limitation there: https://github.com/home-assistant/core/pull/115128#discussion_r1557326343

Reply:
Yeah seems like it can definitely be a problem if the user has a few devices. I think it should be up to the user to decide so I've added it to the config with a default of `600`. Thanks for your review!

Reply:
Oh, nice idea